R/hugo.R

```diff
@@ -92,7 +94,8 @@ change_config = function(name, value) {
 #' if (interactive()) new_site()
 new_site = function(
   dir = '.', install_hugo = TRUE, format = 'toml', sample = TRUE,
-  theme = 'yihui/hugo-lithium-theme', theme_example = TRUE, serve = interactive()
+  theme = 'yihui/hugo-lithium-theme', theme_example = TRUE, serve = interactive(),
+  hostname = 'github.com'
```

Comment:
I think it makes a little more sense to move this argument closer to `theme` (e.g. right after `theme`).

Reply:
will do!